symbols/arm9.yml

```diff
@@ -9998,6 +9998,11 @@ arm9:
         
         r0: Dungeon ID
         return: Dungeon mode
+    - name: SoundUtilGetRandomNumber
```

Comment:
So the sound engine uses a separate RNG function? How interesting. Do we know if it differs from the usual RNG?

Reply:
This is probably a part of the Procyon DSE library rather than the main EoS codebase, which would explain why it's a separate function. It looks like a different implementation, though I haven't traced it far enough to determine what RNG mechanism is being used.